### ISSUE TITLE
Chalkboard: No alt text for images

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#322]
This fails no alt text for images and the more specific rule around using images for text (can't be scaled  with text size increase, or blocky if use page zoom instead).

Issue one:

`<img id="Chalkboard" src="Images/Chalkboard.svg?_ts=1483098742714" style="visibility: visible; width: 260px; left: 230px; top: 138.5px;">`

This should have alt="" role="none". Note this is an SVG with text, but the text is a path rather than ext, and I'm not sure there is any benefit to the screen reader user to reading it.

Issue two:

`<img id="Logo" src="Images/Logo.png?_ts=1483098742714">`

Quick fix would be to include the exact text in the alt attribute, but this would still suffer from the text size not being scalable or translatable etc. 

Better fix would be to just rip it out and use actual text. The image is faded out before the benchmark runs, so I expect there won't be any change to the benchmark characteristics if this is fixed?

Issue three:

`<img id="EmptyChalkboard" src="Images/EmptyChalkboard.svg?_ts=1483098742714?_ts=1483098742714">`

This image is hidden by default and is panned and zoomed when the benchmark is run. 

This could either have an empty alt, but as it is a key part of the benchmark, maybe it has alt text that explains it is an image of a chalkboard that is being scaled, panned, and zoomed by whatever methods are being used.
